### DISCUSSION TITLE
is nuxt 3 good for production now?

### DISCUSSION THREAD

**Discussion by shahensaifullah** [nuxt/nuxt#17597] [Questions]
I am going to build a long-term eCommerce site for my client. I am good with Nuxt and Angular. but I want to build this app with Nuxt 3. Is nuxt 3 good enough for a production-level app now? i see, nuxt 3 stable versions will release next month. If nuxt 3 is not good for production now and if I build an app with nuxt 2, can I migrate easily to nuxt 3 next month? because this app will take 1.5 months to build. Otherwise, I need to build that eCommerce with Angular. I am confused. Someone, please help me.

**Comment by RihanArfan** [comment]
If you're fine potentially dealing with bugs or breaking problems until stable release then I don't see why you couldn't start with Nuxt 3.
